docs/logcli.md

```diff
@@ -26,10 +26,19 @@ Now `logcli` is in your current directory.
 
 ### Example
 
+If you have a hosted Grafana export the variables as below
 ```
 $ export GRAFANA_ADDR=https://logs-us-west1.grafana.net
 $ export GRAFANA_USERNAME=<username>
 $ export GRAFANA_PASSWORD=<password>
+```
+If you are running on local, export the variables as below
```

Comment:
I prefer:
```
Otherwise, when running e.g. [locally](https://github.com/grafana/loki/tree/master/production#run-locally-using-docker), point it to your Loki instance:
```